content/reference/user/profile/index.md

```diff
@@ -9,17 +9,18 @@ weight: 10030
 
 ## Accessing your profile
 
-You can view your profile at any time by clicking on the **My Profile** button in the navigation bar.
+{{< img src="profile_menu.png" alt="My profile button" >}}
 
-{{< img src="profile.png" alt="Profile" >}}
+You can view your profile at any time by clicking on the **My profile** button in the navigation bar.
```

Comment:
There may be a better way to describe where this button is... Do we have a name for the drop-down menu on the organization avatar?

Reply:
on the organization button ar the top-right of the screen ?